pandas/core/groupby/groupby.py

```diff
@@ -2452,12 +2457,28 @@ def ohlc(self) -> DataFrame:
     @doc(DataFrame.describe)
     def describe(self, **kwargs):
         with self._group_selection_context():
+            if len(self._selected_obj) == 0:
+                data = self._selected_obj
+                percentiles = list(refine_percentiles(None))
```

Comment:
I think you need to use kwargs here; otherwise if the user e.g. specifies custom percentiles, you won't get the correct output; is that right?

Reply:
Yes. Updated